### PR TITLE
Get concepts followed by group update

### PR DESCRIPTION
**Pull request by thurstontye**
fixes the issue found in kat-myft-syncher where get concepts followed by group is looking for follwed relationship and not followed_by_kat

### CODE REVIEW

File: test/myFTClient.spec.js [hunk 1]
```diff
@@ -913,6 +914,28 @@ describe('myFT Client proxy', () => {
 					.catch(done);
 			});
 
+			it('Should get an array of concepts followed by a group', done => {
+				if (mockAPI) {
```
Comment (dapenguin) [inline test/myFTClient.spec.js]: Shouldn't need to use the mockAPI flag, so just see if it all still builds when you take it out.

**Review by leggsimon — APPROVED** [review]
LGTM

**Review by dapenguin — APPROVED** [review]
LGTM. Just check to see if you really need to test for the mockAPI flag.

File: test/myFTClient.spec.js (hunk 1 above)
Comment (thurstontye) [reply test/myFTClient.spec.js]: should be please to know mockAPI is gone. At least from the myftClient.spec.js

**Pull request merged**